wikidict/lang/el/__init__.py

```diff
@@ -67,47 +68,34 @@
 
 # Templates to ignore: the text will be deleted.
 templates_ignored = (
+    "λείπει ο ορισμός",
```

Comment:
Is this actually the only relevant line to change? This one, and the one about translating the release (without changing links).

Reply:
Yeah, relevant to https://github.com/BoboTiG/ebook-reader-dict/issues/2248, I also added support for a number of templates at the end.

Reply:
There are too many changes unrelated, and undesired (like one about types, noqa, etc.). Could you open a PR for each specific change? Like one to simply ignore the template, another one to support another template, etc. It will ease the work on our side, and accelerate a potential merge.